homeassistant/components/here_travel_time/sensor.py

```diff
@@ -443,9 +443,9 @@ def _build_hass_attribution(source_attribution: dict) -> str | None:
             return attribution
 
 
-def convert_time_to_isodate(timestr: str) -> str:
-    """Take a string like 08:00:00 and combine it with the current date."""
-    combined = datetime.combine(dt.start_of_local_day(), dt.parse_time(timestr))
+def convert_time_to_isodate(simple_time: time) -> str:
```

Comment:
We should also correct `arrival` and `departure` parameters that are currently typed as `datetime`.

Reply:
This is part of another PR #61398